docs/tutorials/pydy_for_autolev_users.rst

```diff
@@ -0,0 +1,672 @@
+﻿PyDy for Autolev Users
+==========================
+
+Introduction
+----------------
+
+Autolev (now defunct) was a proprietary software which was used for
+symbolic multibody dynamics. The SymPy mechanics module now has enough
+power and functionality to be a fully featured symbolic dynamics module.
+The PyDy package extends the SymPy output to the numerical domain for
+simulation, analyses and visualization. Autolev and PyDy have a lot in
+common but there are also many differences between them. This page shall
+expand upon their differences. It is meant to be a go to reference for
+Autolev users who want to transition to PyDy.
+
+It would be helpful to have a basic idea about scientific computing with
+Python before going over this page. If you aren’t familiar with
+scientific computing with Python there are many sources to learn. You
+can start with the Python programming language itself, with the
+canonical source being the `Python
+Documentation <https://docs.python.org/>`__. The `SciPy
+Lectures <http://www.scipy-lectures.org/>`__ are a great intro to
+scientific computing with Python. Finally, to learn about how to do
+symbolic manipulation with SymPy, check out the `SymPy
+Documentation <http://docs.sympy.org/>`__, especially the tutorial.
+
+Some Key Differences
+------------------------
+
++-----------------------------------+-----------------------------------+
+|          **Autolev**              |             **PyDy**              |            
++===================================+===================================+
+||                                  ||                                  | 
+| Autolev is a language that was    | PyDy is based on Python. While    |
+| designed to perform symbolic      | Autolev’s syntax is more compact  |
+| multibody dynamics. Since it is a | than PyDy’s , PyDy (by virtue of  |
+| language of its own, it has a     | being an add-on to Python) is     |
+| very rigid language               | much more flexible. The users     |
+| specification. It predefines and  | have more control over what they  |
+| computes many things based on the | can do. For example, one can      |
+| input code. Its statements are a  | create a class in their code for  |
+| lot cleaner as a result of this.  | let’s say a type of rigidbodies   |
+|                                   | with common properties.           |
+|                                   |                                   |
+|                                   |                                   |
++-----------------------------------+-----------------------------------+
+||                                  ||                                  |
+| Autolev writes Matlab, C, or      | The PyDy package uses the SymPy   |
```

Comment:
Autolev generates Matlab, C, or Fortran code from a small set of symbolic mathematics.

Reply:
PyDy generates numerical Python, C, or Octave/Matlab code from the large set of symbolic mathematics created with SymPy.